test/unit/fh3/admin/appstore/test_storeitems.js

```diff
@@ -13,7 +13,7 @@ var commands = {
   delgroups: genericCommand(require('cmd/fh3/admin/appstore/storeitems/delgroups')),
   binaryversions: genericCommand(require('cmd/fh3/admin/appstore/storeitems/binaryversions')),
   grouprestrict: genericCommand(require('cmd/fh3/admin/appstore/storeitems/grouprestrict')),
-  list: genericCommand(require('cmd/fh3/admin/appstore/storeitems/list'))
+  list: genericCommand(require('cmd/fh3/admin/appstore/storeitems/list')),
```

Comment:
I guess there you forgot to add line
`add: genericCommand(require('cmd/fh3/admin/appstore/storeitems/add'))`
didn't you? :)

Reply:
👍 tks for the review. The unit test was wrong. It is fixed now. Really tks for your observation.